### ISSUE TITLE
Using Sky causes exception if the user leaves the web page

### ISSUE THREAD

**Issue by clibequilibrium** [threlte/threlte#876]
Using sky in this code raises exception when user navigates back or forward from 3D web page

```svelte
<script>
	import { T, useTask } from '@threlte/core';
	import { OrbitControls, Sky, interactivity } from '@threlte/extras';
	import { spring } from 'svelte/motion';
	import { DEG2RAD } from 'three/src/math/MathUtils.js';
	interactivity();
	const scale = spring(1);
	let rotation = 0;
	useTask((delta) => {
		rotation += delta;
	});
</script>

<Sky elevation={0.5} />

<T.PerspectiveCamera position={[0, 7, 18]} fov={60} near={1} far={20000} makeDefault>
	<OrbitControls
		enableZoom={true}
		maxPolarAngle={85 * DEG2RAD}
		enableDamping
		target={[0, 2.5, 0]}
	/>
</T.PerspectiveCamera>

<T.DirectionalLight position={[0, 10, 10]} castShadow />
<T.Mesh
	rotation.y={rotation}
	position.y={1}
	scale={$scale}
	on:pointerenter={() => scale.set(1.5)}
	on:pointerleave={() => scale.set(1)}
	castShadow
>
	<T.BoxGeometry args={[1, 2, 1]} />
	<T.MeshStandardMaterial color="hotpink" />
</T.Mesh>
<T.Mesh rotation.x={-Math.PI / 2} receiveShadow>
	<T.CircleGeometry args={[4, 40]} />
	<T.MeshStandardMaterial color="white" />
</T.Mesh>

```

![image](https://github.com/threlte/threlte/assets/105135724/60e93f8d-5b17-4622-9dfd-f58be4efee07)

at ``deallocateRenderTarget()``

The user is unable to get back to the 3D scene unless they reload the web page to see 3D scene again. Commenting out ``<Sky>`` fixes the issue.

**Comment by clibequilibrium** [comment]
Possible related to https://github.com/threlte/threlte/issues/804

**Comment by grischaerbe** [comment]
fixed in #877 🎉

**Comment by clibequilibrium** [comment]
> fixed in #877 🎉

ty so much !
